src/SeqViz/Linear/SeqBlock/SeqBlock.tsx

```diff
@@ -1,12 +1,65 @@
 import * as React from "react";
-
-import Annotations from "./Annotations";
+import { MouseEventHandler } from "react";
+import { Annotation } from "../../../part";
+import { SearchResult } from "../../../utils/search";
+import { ICutSite, Primer, SizeType } from "../../Circular/Circular";
+import { SeqVizSelection } from "../../SeqViz";
+import { AnnotationRows } from "./AnnotationRows";
 import CutSiteRow from "./CutSites";
-import IndexRow from "./Index";
 import Find from "./Find";
+import IndexRow from "./Index";
 import Primers from "./Primers";
 import Selection from "./Selection";
-import TranslationRows from "./Translations";
+import TranslationRows, { Translation } from "./Translations";
+
+export type InputRefFuncType = <T>(id: string, ref: unknown) => React.LegacyRef<T>;
+
+export type FindXAndWidthType = (
+  n1: number | undefined,
+  n2?: number
+) => {
+  x: number;
+  width: number;
+};
+
+interface SeqBlockPosition {
+  x: number; // [the x positioning, from left...]
```

Comment:
nit: if you comment above the var or with `/* */` I think VSCode will pick it up and add to the hover-over tips. I don't _think_ that the same happens for inline comments like this (could be wrong)